documentation/book/appendix_metrics.adoc

```diff
@@ -14,89 +14,235 @@ endif::InstallationAppendix[]
+* A `ServiceMonitor` to manage the configuration of the Prometheus pod.
+* A `Secret` to manage additional Prometheus settings.
+* A `PrometheusRule` to manage alert rules for the Prometheus pod.
+* A `Prometheus` to manage the configuration of the Prometheus pod.
+
+==== Deploying Prometheus on {OpenShiftName}
+
+The provided link:https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus.yaml[`prometheus.yaml`^] file, together with the Prometheus related resources, creates a `ClusterRoleBinding` in the `myproject` namespace.
+It also discovers an Alertmanager instance in the same namespace.
+If you are using a different namespace, download the resource file and update it using the following command:
+
+On Linux, use:
+
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus.yaml
+
+On MacOS, use:
+
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus.yaml | sed -e '' 's/namespace: .*/namespace: _my-namespace_/' > prometheus.yaml
 
-[source,shell,subs=attributes+]
-curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/prometheus.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/;s/regex: myproject/regex: _my-namespace_/' > prometheus.yaml
+To define Prometheus jobs that will scrape the metrics data, you must apply the `ServiceMonitor` resource located in the provided `strimzi-service-monitor.yaml` file.
+Download this file using the following command:
 
-The provided `alerting-rules.yaml` file creates a `ConfigMap` with sample alerting rules; download the resource file as follows:
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/strimzi-service-monitor.yaml
 
-[source,shell,subs=attributes+]
-curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/alerting-rules.yaml > alerting-rules.yaml
+Currently, the Prometheus Operator only supports jobs that include an `endpoints` role for service discovery.
+To use another role, edit the `additionalScrapeConfigs` property in the `prometheus.yaml` configuration file.
+This takes the name of the `Secret` and the name of the property in a given `Secret` in which additional configuration is stored.
+To create this `Secret` resource, use the following command:
 
-To deploy all these resources you can run the following.
+[source,shell,subs=+quotes]
+curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/additional-properties/prometheus-additional.yaml
```

Comment:
Same here, the file is not downloaded, but written to STDOUT, I would add a -O (and probably remove the -s) option